app/assets/javascripts/components/involvement_page.es6.jsx

```diff
@@ -0,0 +1,80 @@
+class InvolvementPage extends React.Component {
+
+  renderOption(option) {
+    middle = false;
+    switch (option) {
+      case 'student':
+        optionIcon = ImageConstants.roles.teacher;
+        optionTitle = 'Student';
+        optionDescription = 'Learn your dream instrument at deeply discounted rates.';
+        buttonText = 'Start Learning';
+        break;
+      case 'teacher':
+        optionIcon = ImageConstants.roles.teacher;
+        optionTitle = 'Teacher';
+        optionDescription = 'Share your passion for music with our students and help provide access to affordable music education.';
+        buttonText = 'Start Teaching';
+        middle = true;
+        break;
+      case 'donor':
+        optionIcon = ImageConstants.roles.donor;
+        optionTitle = 'Donor';
+        optionDescription = 'Support Forte in our mission to change the lives of underserved youth musicians.';
+        buttonText = 'Donate Now';
+        break;
+    }
+
+    return (
+      <div className={"option " + (middle ? "middle" : "")}>
+        <div className="optionIcon">
+          <img src={optionIcon} href="#" />
+        </div>
+        <div className="optionTitle">
+          <p> {optionTitle} </p>
+        </div>
+        <div className="separator"/>
+        <div className="optionDescription">
+          <p> {optionDescription} </p>
+        </div>
+        <div className="optionButton">
+          <Button className="button--orange"> {buttonText} </Button>
```

Comment:
1. for buttons, you should always do "button button--orange"
2. remove the spaces between > and { and } and < so 

`<Button className="button button--orange">{buttonText}</Button>`
do it for the other ones too like optionTitle, optionDescription, etc.